src/views/UserScreen.tsx

```diff
@@ -12,6 +14,21 @@ export interface UserScreenProps {
 export default function UserScreen(props: UserScreenProps) {
   const { user } = props;
 
+  const { data, loading, refetch } = useQuery(ME_QUERY, {
+    notifyOnNetworkStatusChange: true,
+    context: {
+      headers: {
+        Authorization: `Bearer ${user.AccessToken}`
+      }
+    },
+    onError: ((error: any) => {
```

Comment:
I think we could probably remove a lot of this console.log logic before checking in